File: templates/grammy/bot.js
// require("dotenv").config();
const { Bot } = require("grammy");

const bot = new Bot(process.env.BOT_TOKEN);

// start command
const start_command = require("./src/commands/start.js");
start_command(bot);

// help command
const help_command = require("./src/commands/help.js");
help_command(bot);

async function start_bot() {
  try {
    console.log("Starting bot. . .");
    await bot.start();
  } catch (err) {
    console.log("An Error Ocurred: Restarting Bot");
    console.log(err);
  }
}

/**********************************
  start bot is for local testing 
*********************************/
start_bot();

File: templates/python/python-telegram-bot/bot.py
from dotenv import load_dotenv
import os
from telegram import Update
from telegram.ext import ApplicationBuilder, CommandHandler
load_dotenv()
from src.commands.help import help
from src.commands.start import start

app = ApplicationBuilder().token(os.getenv('BOT_TOKEN')).build();

app.add_handler(CommandHandler("start", start))
app.add_handler(CommandHandler("help", help))


##################################
# Starting bot for local testing # 
##################################

if __name__ == "__main__":
  try:
    print("Starting bot. . .")
    app.run_polling()
    pass
  except Exception as e:
    print("An Error Ocurred:")
    print(e)